build.gradle

```diff
@@ -22,10 +23,17 @@ subprojects {
 	version = rootProject.version
 	
 	repositories {
+		mavenLocal()
```

Comment:
am not sure if i like that or if we should pass a `maven{url "somepath"}` here

Reply:
too much magic when using a URL here. It is always a bit harder to know to what it actually resolves. The local maven repo is clear to that. The Jenkins script makes sure it us a private one.